### DISCUSSION TITLE
Particles trajectory on land

### DISCUSSION THREAD

**Discussion by Gianlulig** [Parcels-code/Parcels#1741] [Q&A]
### Question

Hi everyone,
I am currently working on a project using Ocean Parcels and have encountered a problem with the trajectories of my particles. Specifically, the particles are incorrectly intersecting with land, which is not the expected behavior.

Below there are some snapshot of my simulation, I think the problem may be in the displacement field? I used the code in the "Preventing stuck particles" tutorial.
Thanks for the help,
Gianluca

![download](https://github.com/user-attachments/assets/aaddf8f6-58ac-4a7d-ad51-72da8abbca54)
![download](https://github.com/user-attachments/assets/c6eb9a44-5493-49cf-9ea7-1384633acaa4)
![download](https://github.com/user-attachments/assets/c9083a36-f508-49b7-9d13-14acaf6c57e6)
![download](https://github.com/user-attachments/assets/ae42a021-40cf-4960-9fd9-1cb9096a3c38)

**Comment by erikvansebille** [comment]
Dear @Gianlulig, thank you for using Parcels and for reaching out here with your question. However, without seeing your code I can't help you. 

Note, more generally, that Parcels by itself is not aware of 'land'; it has no concept of land versus ocean. What to do with land is controlled by the user through Kernels, so when you don't want particles to move on land, that is something that you will have to fix in your custom Kernels.
